src/main/java/seedu/address/logic/commands/student/MedicalHistoryCommand.java

```diff
@@ -65,8 +65,8 @@ public CommandResult execute(Model model) throws CommandException {
         Student editedStudent = new Student(studentToEdit, studentToEdit.getEmergencyContact(),
             studentToEdit.getFormClass(), studentToEdit.getAddress(), medicalHistory);
 
-        model.setPerson(studentToEdit, editedStudent);
-        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+        model.setStudent(studentToEdit, editedStudent);
+        model.updateFilteredStudentList(PREDICATE_SHOW_ALL_STUDENTS);
```

Comment:
we will no longer be showing the medical history within the UI itself, so would it still make sense to show the list of students? There will be no indication that there has been a change

Reply:
I think this just lists all students in the address book, it doesn't tell the address book to show the medical history of students